packages/sources/svelte/src/utils/link.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Compute the rel and add noopener and noreferrer if target are _blank
+ * https://thecodest.co/blog/web-app-security-target-_blank-vulnerability/
+ * @param target
+ * @param rel
+ * @returns {string|*}
+ */
+export const computeRel = (target, rel) =>
+  target === '_blank'
+    ? Array.from(new Set(rel).add('noopener').add('noreferrer'))
+        .join(' ')
+        .trim()
+    : rel;
```

Comment:
Could this be done with Typescript please ? I can't figure out here if `$$restProps['rel']` is returning a string, an array, or an object.

If `rel` is an object, then is not iterable and `new Set(rel)` sould not work.

If `rel` is a string, as it is in other framework, then`new Set(rel)` creates a set of the letters in the string:
```
Array.from(new Set('noopener'))
// ["n", "o", "p", "e", "r"]
```

then using the current script : 
```
computeRel('_blank', 'noopener nofollow')
// "n o p e r   f l w noopener noreferrer"
```

If `rel` is an array, then everything is fine :) (But maybe the arguments could be renamed or typed since there are intuitivelly considered as strings) 

Just FYI, we did something similar here, but with a split method as we were working with strings : https://github.com/Decathlon/vitamin-web/blob/main/packages/sources/react/src/components/actions/VtmnLink/VtmnLink.tsx#L50-L59

Reply:
Nice catch ! It should be 

```
new Set(rel && rel.split(' '))
```

Reply:
This thread is marked as resolve but not fixed yet, is it ?

Reply:
Not fixed yet, I will do that with unit tests :)